model.py

```diff
@@ -0,0 +1,1232 @@
+            self.table_condition = "01380050080"
+        elif self.pn_condition_254 in pn_value:
+            self.table_condition = "19103303700"
+        elif self.pn_condition_255 in pn_value:
+            self.table_condition = "AZ5A74AA801"
+        elif self.pn_condition_256 in pn_value:
+            self.table_condition = "AY5A74AA701"
+        elif self.pn_condition_257 in pn_value:
+            self.table_condition = "11783500000"
+        elif self.pn_condition_258 in pn_value:
+            self.table_condition = "11783500100"
+        elif self.pn_condition_259 in pn_value:
+            self.table_condition = "#85E407271AK"
+        elif self.pn_condition_260 in pn_value:
+            self.table_condition = "11773302701"
+        elif self.pn_condition_261 in pn_value:
+            self.table_condition = "J9C3-3B436-AE"
+        elif self.pn_condition_262 in pn_value:
+            self.table_condition = "#4M0407271T"
+        elif self.pn_condition_263 in pn_value:
+            self.table_condition = "#4MN407271C"
+        elif self.pn_condition_264 in pn_value:
+            self.table_condition = "16GKU9F20"
+        elif self.pn_condition_265 in pn_value:
+            self.table_condition = "11743305400"
+        elif self.pn_condition_266 in pn_value:
+            self.table_condition = "10466758"
+        elif self.pn_condition_267 in pn_value:
+            self.table_condition = "463568890"
+        elif self.pn_condition_268 in pn_value:
+            self.table_condition = "463568900"
+        elif self.pn_condition_269 in pn_value:
+            self.table_condition = "463548020"
+        elif self.pn_condition_270 in pn_value:
+            self.table_condition = "#983501202D"
+        elif self.pn_condition_271 in pn_value:
+            self.table_condition = "105150130"
+        elif self.pn_condition_272 in pn_value:
+            self.table_condition = "#95C501201S"
+        elif self.pn_condition_273 in pn_value:
+            self.table_condition = "1380048080"
+        elif self.pn_condition_274 in pn_value:
+            self.table_condition = "01380045080"
+        elif self.pn_condition_275 in pn_value:
+            self.table_condition = "#4M0501201G"
+        elif self.pn_condition_276 in pn_value:
+            self.table_condition = "CPLA-4K139-EA"
+        elif self.pn_condition_277 in pn_value:
+            self.table_condition = "CPLA-4K138-CA"
+        elif self.pn_condition_278 in pn_value:
+            self.table_condition = "MY42-3D402-AA"
+        elif self.pn_condition_279 in pn_value:
+            self.table_condition = "M8B2-3D402-AA"
+        elif self.pn_condition_280 in pn_value:
+            self.table_condition = "GX53-3N128-AC"
+        elif self.pn_condition_281 in pn_value:
+            self.table_condition = "M8E2-3D402-BB"
+        elif self.pn_condition_282 in pn_value:
+            self.table_condition = "1236350R001"
+        else:
+            self.table_condition = None

```

Comment:
To co uczyniłeś to jest zbrodnia na ludzkości ... nie znam biznes modelu po co jest program ale jak będziesz miał z wcześniejszego kroku obiekt Condition to możesz to zrobić ładniej np.
`class Condition(pedantic.baseModel):
    condition_patern: str
    def validate_condition(self):
          ....
`